docs/advanced/porting_guide.md

```diff
@@ -37,42 +34,43 @@ Add the target description to ```mbed-os\targets\targets.json```:
 }
 ```
 
+See The [mbed Target Documentation](mbed_targets.md) for more details on what this definition means.
+
 ### Bootstrap code
 
-You need a number of files to first successfully compile. You need CMSIS-CORE files for startup and peripheral memory addresses as well as the linker scripts for ARM, IAR and GCC toolchains. These files are usually in the ```mbed-os\targets\TARGET_VENDOR\TARGET_MCU_FAMILY\TARGET_MCUNAME\device``` directory.
+You need a number of files to successfully compile. You need CMSIS-CORE files for startup and peripheral memory addresses, and you need linker scripts for ARM, IAR and GCC toolchains. These files are usually in the ```mbed-os\targets\TARGET_VENDOR\TARGET_MCU_FAMILY\TARGET_MCUNAME\device``` directory.
 
-If the microcontroller has a SYS_TICK, the RTOS will configure this, so only a few macros are needed in ```mbed-os\targets\TARGET_VENDOR\mbed-rtx.h```.
+mbed OS requires dynamic vector relocation, which requires extensions to CMSIS-CORE. Start this process by adding a `mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis.h` file. This header file will include the vector relocation additions and device-specific headers that include CMSIS-CORE. Next add a relocation function in `mbed-os\targets\TARGET_VENDOR\TARGET_MCUNAME\cmsis_nvic.c and .h`. This relocation function will change the contents of the Interrupt Vector Table at run time.
```

Comment:
- Please change "adding a" to "adding an".
- Please change "will include" to "includes" for consistent tense.
- Please change "will change" to "changes" for consistent tense.